### PR TITLE
basic style for alert component

### PR DESCRIPTION
**Pull request by AkihitoVM**

### CODE REVIEW

**Review by mpinter — CHANGES_REQUESTED** [review]
- code styling & newlines at the ends of files
- the styles do not match figma exactly (i.e. check font weight on the buttons or padding on bottom of alert without buttons)

**Review by mpinter — CHANGES_REQUESTED** [review]
The font weights are still incorrect - you should always check the styling against figma:

Screenshot 2022-10-21 at 10 31 14

Screenshot 2022-10-21 at 10 31 37

I.e. the button & the title are wrong. I also see that the gaps (or paddings/margins) between sections inside the alert are wrong - we're being a bit more lenient with these as this can get easier together with fixing the tailwind config (will write regarding this to slack). Still, in case you're doing those 'by eye' for them to roughly match the design, please again - look at the exact numbers in figma and try to follow those.

**Comment by AkihitoVM** [comment]
> The font weights are still incorrect - you should always check the styling against figma:
> 
> Screenshot 2022-10-21 at 10 31 14
> 
> Screenshot 2022-10-21 at 10 31 37
> 
> I.e. the button & the title are wrong. I also see that the gaps (or paddings/margins) between sections inside the alert are wrong - we're being a bit more lenient with these as this can get easier together with fixing the tailwind config (will write regarding this to slack). Still, in case you're doing those 'by eye' for them to roughly match the design, please again - look at the exact numbers in figma and try to follow those.

Ako som spominal na calle, typ takeho buttonu nie je implementovany v ramci Buttonov vo figme, je to iba vzorec vyuzitia tlačitka.

File: next/tailwind.config.js [hunk 1]
```diff
@@ -270,6 +300,9 @@ module.exports = {
         61: '244px',
         87: '350px',
       },
+      gap: {
+        3: '13px'
```
Comment (mpinter) [inline next/tailwind.config.js:304]: still not convinced about this one but let's deal with it in the global spacing rewrite (will create a JIRA task today)

**Review by mpinter — APPROVED** [review]
:+1: lgtm

**Pull request merged**